samples/getting-started/README.md

```diff
@@ -8,4 +8,6 @@
 - **[Simple Algorithms](./simple-algorithms/)**:
   This sample covers several different basic quantum algorithms, and how each can be written in Q#.
 - **[Intro to IQ#](./intro-to-iqsharp/)**:
-  This samples explains how to create interactive notebooks using Jupyter and IQ#.
+  This sample explains how to create interactive notebooks using Jupyter and IQ#.
+- **[Azure Quantum services](./azure-quantum)**:
```

Comment:
I'm not sure about this one? The whole samples repo is effectively an intro to Azure Quantum since that term is inclusive of the Quantum Development Kit as well as services, whereas this folder is more narrowly scoped to the service component. Perhaps this?

```suggestion
- **[Intro to Azure Quantum services](./azure-quantum)**:
```

As a side thought, the name of this folder is a bit historical, but eventually we should consider renaming to make the scoping a bit more clear

Reply:
I'm going off of what we say in our docs:
> Azure Quantum is a cloud service of quantum partners and technologies for running quantum computing programs or solving optimization problems. 

It looks like users would expect "Azure Quantum" to mean our cloud services. I'm not sure we will want to use internal jargon here.

See https://docs.microsoft.com/en-us/azure/quantum/overview-azure-quantum#:~:text=Azure%20Quantum%20is%20a%20cloud%20service%20of%20quantum,running%20on%20classical%20Azure%20hardware%20for%20immediate%20results.

Reply:
I don't think it's internal jargon at all, quite the opposite? All of the Quantum Development Kit, Q#, all of our libraries, client packages, are documented as being parts of "Azure Quantum," including being hosted at docs.microsoft.com/azure/quantum. Being explicit here helps avoid relying on internal jargon and helps to disambiguate.

Reply:
I agree, there is a discrepancy here. It looks like our docs are indeed focused on Azure Quantum-only, which makes it look like the QDK is part of Azure Quantum.